src/components/Cropper/index.js

```diff
@@ -51,7 +51,7 @@ class Cropper extends React.Component {
     this.croppie
       .result({
         type: 'blob',
-        size: 'viewport',
+        size: { width: 300, height: 300 },
```

Comment:
@villanuevawill let me know what size you want this to be -- alternatively i could set this to `original` so it just applies the crop to the original size of the image (seems like it could be bad if they upload a large image